.travis.yml

```diff
@@ -16,9 +16,12 @@ addons:
     sources:
       - sourceline: 'ppa:chris-lea/libsodium'
       - sourceline: 'ppa:maarten-fonville/protobuf'
+      - sourceline: 'ppa:exonum/rocksdb'
     packages:
       - build-essential
       - libsodium-dev
+      - librocksdb5.17
```

Comment:
Shall we specify a particular verison?

Reply:
This is the package name, so yes

Reply:
Why not librocksdb-dev with no version?

https://launchpad.net/~exonum/+archive/ubuntu/rocksdb/+sourcepub/9938499/+listing-archive-extra

Reply:
The packages are named according the Debian/Ubuntu naming convention

Reply:
This is the package from our ppa, not ubuntu repos

Reply:
@vitvakatu  If there is librocksdb-dev present in our PPA, why don't we use that?

Reply:
The Ubuntu package building tool (debuild) uses this convention when building packages

Reply:
🤔 Because -dev contains headers, and we need only dynamic library to link with?

Reply:
I see, that makes sense, thanks. But is it OK to fix the version (won't we forget to update it)?

Reply:
Not sure, but I think it's okay for now. Anyway, we'll need to test new version before actually using it. And not locking the version may accidentally break our code after update

Reply:
:ok_man: , does the core have an issue to update the docs with Rocksdb installation instructions?

Reply:
Would you please add a changelog entry that Rocksdb from our PPA must be installed on linux?